azure/util.py

```diff
@@ -0,0 +1,324 @@
+                zip_file_object.extract(member, '.')
+                break
+
+
+"""
+ Helper function to upload to AzureBlob storage based on
+ Storage account,
+ Storage container,
+ SAS Token
+"""
+
+
+def upload_to_azure_blob(args):
+    if os.name == "nt":
+        download_azcopy_script_for_windows()
+        commands = ["azcopy copy \"{}\" \"https://{}.blob.core.windows.net/{}{}\" --recursive=True"
+                        .format(args.source_path,
+                                args.account_name,
+                                args.container_name,
+                                args.sas_token)]
+        run_commands(commands)
+    else:
+        download_azcopy_script()
+        commands = ["./azcopy copy \'{}\' \'https://{}.blob.core.windows.net/{}{}\' --recursive=True"
+                        .format(args.source_path,
+                                args.account_name,
+                                args.container_name,
+                                args.sas_token)]
+        run_commands(commands)
+    configure_yaml_storage(args.container_name)
+
+
+"""
+ Sets KUBECONFIG environment variable to 
+ the path to the  json file generated
+"""
+
+
+def set_kubeconfig_environment_var():
+    if os.path.isdir("_output"):
+        if os.name == "nt":
+            config_path = PETCTL_DIR + "\\_output\\azure-pytorch-elastic\\kubeconfig"
+        else:
+            config_path = PETCTL_DIR + "/_output/azure-pytorch-elastic/kubeconfig"
+        logger.info("Reading KUBECONFIG environment variable from {}".format(config_path))
+
+        for files in walk(config_path):
+            for f in files:
+                if f and f[0].endswith(".json"):
+                    if os.name == "nt":
+                        config_path = config_path + "\\" + f[0]
+                    else:
+                        config_path = config_path + "/" + f[0]
+
+        if config_path.endswith(".json"):
+            os.environ["KUBECONFIG"] = config_path
+            logger.info("Setting KUBECONFIG env variable {}".format(os.environ.get("KUBECONFIG")))
+
+
+# Create storage secret named 'pet-blob-secret'
+def create_storage_secrets(args):
+    commands = [
+        "kubectl create secret generic pet-blob-secret \
+                 --from-literal accountname={0} \
+                 --from-literal accountkey={1} \
+                 --type=azure/blobfuse".format(
+            args.account_name, args.account_key
+        )
+    ]
+    run_commands(commands)
+
+
+# Install Azure blobfuse drivers
+def install_blobfuse_drivers():
+    commands = [
+        "kubectl apply -f https://raw.githubusercontent.com/Azure/kubernetes-volume-drivers/master/flexvolume/blobfuse/deployment/blobfuse-flexvol-installer-1.9.yaml"
+    ]
+    run_commands(commands)
+
+
+# Create docker image secrets given user inputs
+def create_docker_image_secret(args):
+    configure_yaml_docker(args.image_name)
+    commands = [
+        "kubectl create secret \
```

Comment:
Minor: you can use Python 3.6 format strings like this:
```
f"""...
       --docker-server={args.server}
       --docker-username={args.username}
       --docker-password={args.password}
"""
```

Reply:
Done, thanks!

Reply:
@drdarshan Does the PR look good to go or are we waiting on some other changes?

Reply:
Hello @aashna - yes, this looks good to me! I'll take care of the merge. Thanks again for this contribution!

Reply:
Thank you @drdarshan :-)